ui/package.json

```diff
@@ -13,7 +13,8 @@
     "reinstall": "npm run clean && rm -rf **/node_modules && npm install",
     "clear-turbo-cache": "rimraf node_modules/.cache/turbo",
     "e2e": "turbo run e2e",
-    "e2e:debug": "turbo run e2e -- --debug"
+    "e2e:debug": "turbo run e2e -- --debug",
```

Comment:
Really not married to this idea, but wonder if we want to be able to write this as `e2e: turbo run e2e:local`? So it's easy to tell that one command is running local configs and one command is running CI configs? 

Still have the convenience of just `npm e2e`, but when peering at the details in package.json it might be more obvious what the difference is?

Again, I think it's fine as is, and this might be a strange idea, b/c I don't do as much test config work.

Reply:
I think that's a good idea to increase clarity! Will fix it up.